hazelcast/src/main/java/com/hazelcast/map/impl/operation/EntryOperation.java

```diff
@@ -407,6 +411,22 @@ private void executeMutatingEntryProcessor(final Object oldValue, String executo
             }
         }
 
+        private void doExecute(String executorName, Runnable runnable) {
+            boolean statisticsEnabled = mapContainer.getMapConfig().isStatisticsEnabled();
+            ExecutorStats executorStats = mapServiceContext.getOffloadedEntryProcessorExecutorStats();
+            try {
+                Runnable command = statisticsEnabled
+                        ? new StatsAwareRunnable(runnable, executorName, executorStats) : runnable;
+                executionService.execute(executorName, command);
+            } catch (RejectedExecutionException e) {
+                if (statisticsEnabled) {
+                    executorStats.rejectExecution(executorName);
+                }
+
+                throw ExceptionUtil.rethrow(e);
```

Comment:
Again, can this wrap the exception that was previously not wrapped?

Reply:
removed wrapping.